src/app/parcels/ActionBar/Actions.tsx

```diff
@@ -29,32 +31,44 @@ export type ActionName =
     | "Download Driver Overview"
     | "Delete Parcel";
 
-type AvailableActionsType = {
-    [actionKey in ActionName]: {
-        newStatus: StatusType;
-    };
+type ActionTypes = {
+    actionName: ActionName;
+    newStatus: StatusType;
+    availableToRole: UserRole[];
 };
 
-export const availableActions: AvailableActionsType = {
-    "Download Day Overview": {
+const availableActions: ActionTypes[] = [
+    {
+        actionName: "Download Day Overview",
         newStatus: "Day Overview Downloaded",
+        availableToRole: ["volunteer", "staff", "manager", "admin"],
```

Comment:
Is there an exported array of all roles anywhere? If so, could we use that for the ones that are available to all roles -just to make it easier if a new role is added? Happy to leave if there isn't.

Reply:
Not sure if we're talking about the same thing here but I agree that we should have some role groups, e.g.
```
everyone = ["volunteer", "staff", "management", "admin"]
org = ["staff", "management", "admin"]
adminOnly = ["admin"]
``` 
and put those in a file somewhere in case we suddenly have many roles and many groups

Wonder if we should make it configurable in the db somewhere (e.g. a new `roles` table) rather than hardcoding 🤔 both the lists and which list is assigned to each action, although that sounds like extra UI work so might be better to leave it and add if requested, only laying the foundations here

I think having groups like that would fix the problem of "we made a new role and forgot to add it to an action" - you'd have to forget the entire group

Reply:
Yep! That was what I was talking about, happy with adding a list to a separate file like the above. There is another point in the codebase where we go allUserRoles = [..] so commonising that would be nice. The db option would probably be better eventually but probably out of scope for this ticket?

Reply:
Thanks both - I've put the lists in roles.tsx and replaced allUserRoles with the new list.